manifests/integreatly-monitoring/monitoring-1.1.4/application-monitoring-operator.v1.1.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,360 @@
+                  initialDelaySeconds: 4
+                  periodSeconds: 10
+                resources: {}
+              serviceAccountName: application-monitoring-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - serviceaccounts
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - deployments/finalizers
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - alertmanagers
+          - prometheuses
+          - prometheusrules
+          - servicemonitors
+          verbs:
+          - '*'
+        - apiGroups:
+          - applicationmonitoring.integreatly.org
+          resources:
+          - applicationmonitorings
+          - applicationmonitorings/finalizers
+          - blackboxtargets
+          - blackboxtargets/finalizers
+          verbs:
+          - '*'
+        - apiGroups:
+          - integreatly.org
+          resources:
+          - grafanadatasources
+          - grafanadatasources/status
+          - grafanadashboards
+          - grafanadashboards/status
+          - grafanas
+          - grafanas/status
+          - grafanas/finalizers
+          - grafanadatasources/finalizers
+          - grafanadashboards/finalizers
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          - routes/custom-host
+          verbs:
+          - '*'
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - rolebindings
+          - roles
+          verbs:
+          - '*'
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - '*'
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: application-monitoring-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - Prometheus
+  - Grafana
+  - Monitoring
+  - AlertManager
+  maintainers:
+  - email: placeholder@email.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: application-monitoring-operator.v1.1.2
```

Comment:
I think this needs to be removed, we do not have this field in any other CSV

Reply:
We actually do keep it in the last manifest: https://github.com/integr8ly/integreatly-operator/blob/master/manifests/integreatly-monitoring/monitoring-1.1.2/application-monitoring-operator.v1.1.2.clusterserviceversion.yaml#L347

I think we can do this now because we aren't deploying through operator hub

Reply:
Ok ill remove that. Why is it created by the make gen/csv if its not needed?

Reply:
@ciaranRoche Just checked with Oleg we do need it